03_Modules/Locations/src/module/api.ts

```diff
@@ -138,4 +154,24 @@ export class LocationsModuleApi
       Number(connectorId),
     );
   }
+
+  /**
+   * Admin Endpoints
+   **/
+
+  // TODO add "admin endpoint" annotation for "regular" error handling when available
+  @Put('/admin')
+  @ResponseSchema(OcpiEmptyResponse, {
+    statusCode: HttpStatus.OK,
+    description: 'Successful response',
+  })
+  async createLocation(
+    @QueryParam('broadcast') broadcast: boolean,
```

Comment:
it's required so that the endpoint validation can stop people from creating locations and wondering why their broadcasts aren't going to MSP if it silently defaults to false. 

if we're not worried this is a problem where people are wondering why the broadcasts aren't happening if they didn't set broadcast true, i can make this an optional field and default it to false 🤔